onfido/data.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import json
+
+
+def get_countries() -> list:
+    """Return a list of countries for consumer parsing."""
+    with open("onfido/data/supported_applicant_countries.json") as f:
+        countries = json.load(f)
+    return countries
+
+
+def check_supported_country(alpha3: str) -> bool:
+    """Check whether a given country is supported by OnFido."""
+    with open("onfido/data/supported_applicant_countries.json") as f:
+        countries = json.load(f)
```

Comment:
This should just call `get_countries`:
```suggestion
    countries = get_countries()
```